src/errorOut.js

```diff
@@ -1,19 +0,0 @@
-"use strict";
-
-var logError = require("./logError");
-var { FirebaseError } = require("./error");
-
-module.exports = function(client, error) {
```

Comment:
So to be clear it looks like `client` was totally unused?

Reply:
yuuuup